integrations/pagerduty/.port/resources/port-app-config.yaml

```diff
@@ -10,6 +10,8 @@ resources:
           blueprint: '"pagerdutyService"'
           properties:
             status: .status
+            url: .html_url
+            oncall: .oncall_user[0].user.email
```

Comment:
Can you verify that if the oncall user is empty it's inserting null?

Reply:
This is one of the blueprint entity data when the oncall_user is unavailable

```json
{
  "identifier": "PWJAGSD",
  "title": "Port Internal Service Updated",
  "blueprint": "pagerdutyService",
  "team": [],
  "properties": {
    "status": "critical",
    "url": "https://getport-io.pagerduty.com/service-directory/PWJAGSD"
  },
  "relations": {},
  "createdAt": "2023-08-07T07:56:01.687Z",
  "createdBy": "hBx3VFZjqgLPEoQLp7POx5XaoB0cgsxW",
  "updatedAt": "2023-08-07T08:14:54.954Z",
  "updatedBy": "hBx3VFZjqgLPEoQLp7POx5XaoB0cgsxW"
}

```

Reply:
Also, we see that even when the oncall_user is empty, the jq mapping returns null

![image](https://github.com/port-labs/Port-Ocean/assets/15999660/bd2c8511-4115-4c10-bd93-a0233a729a3d)